src/internal.c

```diff
@@ -2194,14 +2194,10 @@ void SSL_CtxResourceFree(WOLFSSL_CTX* ctx)
             ctx->x509_store.objs = NULL;
         }
     #endif
-    #ifdef OPENSSL_EXTRA
+    #if defined(OPENSSL_EXTRA) || defined(WOLFSSL_EXTRA) || defined(HAVE_LIGHTY)
```

Comment:
What is `WOLFSSL_EXTRA`? I can't find it in configure.ac. Looks like it's only used to gate in compilation of `ca_names` in internal.h. Would be good to see if this macro can be removed.

Reply:
Still wondering about this.

Reply:
Oh, I'm not sure where `WOLFSSL_EXTRA` is coming from. It's how `ca_names` is defined in the `WOLFSSL_CTX` struct so it should be `free`'d in the same conditions. I'll ask if anyone knows anything about this define.

Reply:
For future readers: 8d0731a5d83e902d1041b521d776a3f90fec844c

Reply:
This looks like the right macro guard for `ctx->ca_names`, but not `ctx->x509_store_pt`. `ctx->x509_store_pt` exists only if `#if defined(OPENSSL_EXTRA) || defined(HAVE_WEBSERVER) || defined(WOLFSSL_WPAS_SMALL)`.

Reply:
Changing guard.